README.md

```diff
@@ -3,12 +3,72 @@
 [![Build Status](https://api.travis-ci.com/maxmind/GeoIP2-node.svg?branch=master)](https://travis-ci.com/maxmind/GeoIP2-node)
 [![Known Vulnerabilities](https://snyk.io/test/github/maxmind/GeoIP2-node/badge.svg)](https://snyk.io/test/github/maxmind/GeoIP2-node)
 
-## Database Reader
+## Description
+
+This package provides an API for the [GeoIP2
+databases](https://dev.maxmind.com/geoip/geoip2/downloadable), and will
+provide an API for the [GeoIP2 Precision web
+services](https://dev.maxmind.com/geoip/geoip2/web-services).  This API also
+works with MaxMind's free [GeoLite2
+databases](https://dev.maxmind.com/geoip/geoip2/geolite2/).
+
+## Installation
+
+```
+yarn add geoip2-node
+```
+
+If you are not able to use `yarn`, you may also use `npm`:
+
+```
+npm install geoip2-node
+```
+
+
+## IP Geolocation Usage
+
+IP geolocation is inherently imprecise. Locations are often near the center of
+the population. Any location provided by a GeoIP2 database or web service
+should not be used to identify a particular address or household.
+
+## Database Usage
+
+The database reader returns a promise that resolves with a reader instance.
+You may call then call the function corresponding to the request type (e.g.
+`city` or `country`), passing it the IP address you want to look up.
+
+If the request succeeds, the function call will return an object for the GeoIP2
+lookup. The object in turn contains multiple record objects, each of which
+represents part of the data returned by the database.
+
+### Options
+
+We use the [node-maxmind](https://github.com/runk/node-maxmind) library as the
+database reader. As such, you have access to the same
+[options](https://github.com/runk/node-maxmind#options) found in that library
+and can be used like this:
+
+```
+const reader = require('geoip2-node').Reader;
+// Typescript:
+// import { Reader } from 'geoip2-node';
+
+const options = {
+  // you can use options like `cache` or `watchForUpdates`
+};
+
+Reader.open('/usr/local/database.mmdb', options).then(reader => {
+  reader.country('1.1.1.1');
```

Comment:
Maybe just to make it a bit more complete..
```
const response = reader.country('1.1.1.1');
...
```
or `console.log(reader.country('1.1.1.1'));`